cloud-functions/functions/custom-logo.js

```diff
@@ -0,0 +1,50 @@
+// this is meant to be run by Covid Watch devs to set a custom logo for different organizations to appear in the NavBar
+// perhaps at some point this could be adapted for an admin user to set their organization's logo
+
+const admin = require('firebase-admin');
+const fs = require('fs');
+
+// these desktop paths for testing purposes, they work!  simulating a dev downloading a file from browser, saving to desktop, then referencing in this script to upload
+const soylentLogoPath = '../../../../soylent-green-logo.png'; // https://fanart.tv/fanart/movies/12101/hdmovielogo/soylent-green-51744f6296779.png
+const pepsiPath = '../../../../pepsi-logo.png'; //https://upload.wikimedia.org/wikipedia/commons/thumb/0/0f/Pepsi_logo_2014.svg/1200px-Pepsi_logo_2014.svg.png
+
+// from some tests, svgs and jpegs don't work. line 39 hardcodes to png based base64 so that may be part of it
+const uOfAPath = '../../frontend/client/assets/uofa-logo.svg';
+const jpegExample = '../../../../Jesse_Colligan.jpeg';
+
+// note: environment should be manually set to "dev", "staging", or "production", depending on where you would like to update the logo
+const setLogo = (environment, organizationID, logoPath) => {
+  // Initialize admin SDK
+
+  // note: only the dev-firebase-admin keys are found in this directory.  staging and prod are elsewhere so this path is not correct for staging and prod
```

Comment:
This isn't true, its up to the user to put the keys in this directory. If these dev keys are committed to the repo, we need to revoke them.

Reply:
Yup.  I believe I only have the dev json locally, I don't have the staging or prod files currently.  So will need the staging one in order to test further in staging before attempting to adjust in prod.